### ISSUE TITLE
[Bug]: x-date component return key bug

### ISSUE THREAD

**Issue by putera** [tasteui/tasteui#725]
### What Happened

When i use ```<x-date />``` and hit "Enter/Return key" it will reset the date to startOfMonth date. Like the picture given.

![](https://i.ibb.co/34P9trB/enter-error-date.gif)

### Component

Date

### How to Reproduce

1. Install fresh laravel.
2. Install ```wire-element``` from [github.com/wire-element modal](https://github.com/wire-elements/modal)
3. add ```x-date``` and ```x-input``` into the modal.
4. hit enter on the x-input

```blade
<form wire:submit="save">
            <x-date />
            <x-input />

           <x-button type="submit"/>
</form>
```

### Laravel Version

10.48.22

### Livewire Version

3.5.11

### TallStackUI Version

1.35.9

### PHP Version

8.2.15

### Operation System

macOS

**Comment by joelhmetz** [comment]
Can you show the code like this?

**Comment by devajmeireles** [comment]
@putera any updates?

**Comment by putera** [comment]
> Can you show the code like this?

I already post the code. Like this? Like what ?

**Comment by putera** [comment]
> > Can you show the code like this?
> 
> I already post the code. Like this? Like what ?

Here the sample repo.

1. Clone the repo, https://github.com/putera/ts-wire-element
2. Run ```composer install && npm run dev```
3. Click the "Open Modal" button
4. When the dialog open, select the date of the calendar, and put any text on the next input field. Press ENTER key. The submit button not trigger and the xdate will return the 1st calendar day.

Please help. Thanks!

**Comment by devajmeireles** [comment]
Thanks to @putera who fixed the issue in #746. Released in v1.36.1 just now!
